_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -32,9 +32,7 @@ This rule applies to any [WAI-ARIA state or property][] that is specified on an
 
 ## Expectation
 
-Each test target is either an [inherited][], [supported][], or [required][] [state][] or [property][] of the [semantic role][] of the element on which the attribute is specified. If the element has no [semantic role][], the attribute must be a [global state or property][global].
-
-**Note:** Assessing the value of the attribute is out of scope for this rule.
+Each test target is either an [inherited][], [supported][], or [required][] [state][] or [property][] for the element's [semantic role][] or the [WAI-ARIA state or property][] may be used on the language feature. If the element has no [semantic role][], the attribute must be a [global state or property][global].
```

Comment:
I'm a bit uneasy in leaving "language feature" as it and only explain it in a background note (versus trying to have a definition).
The rule is only targeting HTML and SVG, and I don't think there is a similar "ARIA in SVG" (or other feature document) out there. So maybe we can be a bit more prescriptive here (to make sure and remove the ambiguity). Pointing here to ARIA in HTML and saying that if the target is an HTML element, it may also have stuff described there.

Reply:
Yes, I thought about that too. In the end, I left the "language feature" so that everybody else can say what they think about it. It's unlikely, I think, there will be an SVG accessibility API mappings spec so we could tighten it.

Reply:
Yes, I thought about that too. In the end, I decided to leave the "language feature" for everybody else to decide on it. Yeah, I'm not sure if there ever will be an equivalent concept for SVG so we can tighten it.

Reply:
I think you're trying to pack too much information into one sentence. It might be better to break this up into a list:

```suggestion
One of the following is true for each test target:

- **Global**: the test target is a [global state or property][global]; or

- **Semantic Role**: the test target is an [inherited][], [supported][], or [required][] [state][] or [property][] of the [semantic role][] of the element on which the test target is specified; or

- **language feature**: the test target is allowed as a language feature on the element on which it is specified. Which ARIA state or property may be used on which element is describe in [ARIA in HTML][].
```

This will also make it easier to add prohibited attributes when ARIA 1.2 makes it to rec.